lnst/Controller/ContainerPoolManager.py

```diff
@@ -73,6 +76,20 @@ def image(self, name):
 
         self._image = name
 
+    @staticmethod
+    def _import_optionals():
+        try:
+            global APIError
+            global Container
+            global Network
+
+            from podman.errors import APIError
+            from podman.domain.containers import Container
+            from podman.domain.networks import Network
+
+        except ModuleNotFoundError as e:
+            not_imported_error(e, PoolManagerError)
```

Comment:
shouldn't we just make a `LnstDependencyError` exception class that would take a list of module names as parameter and format a stable error message? wouldn't that work better than a method that first prints end then raises a generic exception?

@jtluka what do you think since the method approach was your proposal I think?

Sorry I didn't comment on it earlier I just noticed that suggestion...

Reply:
Yeah, your proposal would look better.

Reply:
I was thinking about that, but not sure. If there is a try-except block that handles `TRexError` it would suddenly crash

Reply:
Discussed offline